lib/elixir/lib/module/types.ex

```diff
@@ -0,0 +1,850 @@
+defmodule Module.Types do
+  @moduledoc false
+
+  defmacrop is_var(expr) do
```

Comment:
Can't you use `defguardp`?

Reply:
It seems to not be available during bootstrap.